internal/app/service/service_handler_test.go

```diff
@@ -109,6 +111,96 @@ var _ = Describe("When listing Services", Label("app", "ListServices"), func() {
 			Entry("When  pageSize is 10 and the database was returning 11 elements", 10, 11, 10, true),
 		)
 	})
```

Comment:
If we want to enforce the above with checking that the two counts where giving back results for the same set of services we should also add a test for that